setup.py

```diff
@@ -38,7 +38,13 @@
 # Add S3QL sources
 sys.path.insert(0, os.path.join(basedir, 'src'))
 sys.path.insert(0, os.path.join(basedir, 'util'))
-import s3ql
+
+# Get S3QL Version & Release from __init__.py
+with open('src/s3ql/__init__.py', 'r') as init_file:
+    init_content = init_file.read()
```

Comment:
Best practice is to parse line by line, then you also don't need the MULTLINE flag:

```
for line in init_fh:
   if re.match('^VERSION...', line):
       # ...
       break
```

Reply:
Okay - I thought the single multiline regex would be simpler/shorter. Done.